Adapter.C

```diff
@@ -10,7 +10,7 @@ preciceAdapter::Adapter::Adapter(const Time& runTime, const fvMesh& mesh)
 : runTime_(runTime),
   mesh_(mesh)
 {
-    adapterInfo("Loaded the OpenFOAM-preCICE adapter v1.0.0.", "info");
+    adapterInfo("Loaded the OpenFOAM-preCICE adapter - unreleased version.", "info");
```

Comment:
Not sure if unreleased is correct. In the end, it is still `v1.0.0` as nothing changed. If we really want to go up here we should maybe select something like `v1.1.0-pre`. However, you know this probably better than me so I will approve. 
(History looks indeed very messy)

Reply:
I forgot about this change, thanks! I guess this really raises the question on whether we should actually release a new version, then, even if we don't have functional changes. But a version number should only refer to a very specific and unique state of the code.

Maybe there is some bug fix we want to fit in, as a workaround? :laughing:

Reply:
I can't find it right now, but we had a discussion where we came to a conclusion on how to treat new OpenFOAM releases, right?

Reply:
Here is the discussion: https://github.com/precice/openfoam-adapter/issues/52

I have already released an additional archive mentioning v2106 in the name.

The strange situation now is that we only released two months ago and there has been no functionality-related development since.

Reply:
Decided to go for a `+ unreleased changes` statement.